platform/src/lib.rs

```diff
@@ -125,4 +134,11 @@ pub trait Platform {
     ///
     /// Example: 99991231235959Z is December 31st, 9999 23:59:59 UTC
     fn get_cert_validity(&mut self) -> Result<CertValidity, PlatformError>;
+
+    /// Retrieves the SubjectAlternativeName extension
+    ///
+    /// Currently, only the otherName choice is supported. This function
+    /// can be left unimplemented if the SubjectAlternativeName extension is
+    /// not needed in the DPE leaf cert or CSR.
+    fn get_subject_alternative_name(&mut self) -> Result<OtherName, PlatformError>;
```

Comment:
nit: Should this return a generic SubjectAltName enum where once of the options is OtherName? Then if we want to support the other types we don't need to change this API.

Reply:
Done